bpython/pager.py

```diff
@@ -30,29 +32,29 @@
 import shlex
 
 
-def get_pager_command(default="less -rf"):
+def get_pager_command(default: str = "less -rf") -> list[str]:
     command = shlex.split(os.environ.get("PAGER", default))
     return command
 
 
-def page_internal(data):
+def page_internal(data: str) -> None:
     """A more than dumb pager function."""
     if hasattr(pydoc, "ttypager"):
         pydoc.ttypager(data)
     else:
         sys.stdout.write(data)
 
 
-def page(data, use_internal=False):
+def page(data: str, use_internal: bool = False) -> None:
     command = get_pager_command()
     if not command or use_internal:
         page_internal(data)
     else:
         curses.endwin()
         try:
             popen = subprocess.Popen(command, stdin=subprocess.PIPE)
```

Comment:
For me this needs a
```suggestion
            assert popen.stdin is not None
            popen = subprocess.Popen(command, stdin=subprocess.PIPE)
```
to silence mypy errors about stdin maybe being None.